lib/actions/device.coffee

```diff
@@ -293,6 +308,71 @@ exports.hasDeviceUrl =
 			console.log(hasDeviceUrl)
 		.nodeify(done)
 
+exports.getApplicationInfo =
+	signature: 'device application info <uuid>'
+	description: 'get application container information'
+	help: '''
+		Use this command to get application container information from device
+
+		Examples:
+
+			$ resin device application info 23c73a1
+	'''
+	permission: 'user'
+	action: (params, options, done) ->
+		resin = require('resin-sdk-preconfigured')
+		resin.models.device.getApplicationInfo(params.uuid).then (appInfo) ->
+			console.log(appInfo)
+		.nodeify(done)
+
+exports.startApplication =
+    signature: 'device start application <uuid>'
```

Comment:
I'd rather this the sub-command here was a single word, rather than sort-of feeling like a namespace here (what else can I `start`?). `resin device start-app` sounds good to me.

Oooh, in fact, better actually: `resin device app start` (to fit with `resin device app info` above), and we can nest all device-specific application commands there.